test_sift_verification_apis.py

```diff
@@ -0,0 +1,66 @@
+from decimal import Decimal
+import sift
+import sys
+
+if sys.version_info[0] < 3:
```

Comment:
it is okay for us to concentrate only on Python 3 in these tests so feel free to remove all that is not related to Pythion 3